CONTRIBUTING.adoc

```diff
@@ -66,9 +66,9 @@ export OS_OUTPUT_GOPATH=1
 
         $ mkdir -p $GOPATH/src/github.com/openshift
         $ cd $GOPATH/src/github.com/openshift
-        $ git clone git://github.com/<forkid>/origin  # Replace <forkid> with the your github id
+        $ git clone https://github.com/<forkid>/origin  # Replace <forkid> with the your github id
```

Comment:
Got it.
Please check another issue:
2. Fix a link for docker machine which can be used to access accurately the content in the document.

 docs/cluster_up_down.md

 -- [Docker Machine](#docker-machine-details)
 +- [Docker Machine](#docker-machine)

@liggitt  Thanks!